request.go

```diff
@@ -203,6 +204,14 @@ func (r *Request) WithName(name string) *Request {
 	opChain := r.chain.enter("WithName()")
 	defer opChain.leave()
 
+	if opChain.failed() {
+		return r
+	}
+
+	if !r.checkOrder(opChain, "WithName") {
```

Comment:
When we pass function names to enter(), setType(), setBody(), etc, we use form `"Foo()"` instead of `"Foo"`.

For consistency, could you please adjust checkOrder calls accordingly? So that

```
r.checkOrder(opChain, "WithName")
```

will be replaced with

```
r.checkOrder(opChain, "WithName()")
```

and so on.